mlir/lib/Dialect/MIOpen/Transforms/AffineTransforms.cpp

```diff
@@ -11,6 +11,7 @@
 
 #include "llvm/ADT/MapVector.h"
 #include "llvm/Support/raw_ostream.h"
+#include <set>
```

Comment:
please switch to `SmallSet`
https://llvm.org/doxygen/classllvm_1_1SmallSet.html